### PR TITLE
fix for kubrenetes name (DNS-1123)

### LINKED ISSUE #2393: Irragular nuclio function name on kubernetes
I try to deploy nulcio funcions on kubernetes via [deply.sh](https://github.com/openvinotoolkit/cvat/blob/develop/serverless/deploy.sh)

But it will occur the DNS-1123 error cause the function name in the function.yaml

```
Error - NuclioFunction.nuclio.io "openvino.omz.public.faster_rcnn_inception_v2_coco" is invalid: metadata.name: Invalid value: "openvino.omz.public.faster_rcnn_inception_v2_coco": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
    /nuclio/pkg/platform/kube/deployer.go:100
```

### PR DESCRIPTION
**Pull request by xuchenCN**
Fix #2393

### CODE REVIEW

File: serverless/openvino/omz/public/mask_rcnn_inception_resnet_v2_atrous_coco/nuclio/function.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # have enough memory (more than 4GB). Look here how to do that
 # https://stackoverflow.com/questions/44417159/docker-process-killed-with-cryptic-killed-message
 metadata:
-  name: openvino.omz.public.mask_rcnn_inception_resnet_v2_atrous_coco
+  name: openvino-mask-rcnn-inception-resnet-v2-atrous-coco
```
Comment (azhavoro) [inline serverless/openvino/omz/public/mask_rcnn_inception_resnet_v2_atrous_coco/nuclio/function.yaml:5]: could you keep `omz-public` in the name?

**Review by azhavoro — COMMENTED** [review]
Thanks for the contribution!

**Review by nmanovic — APPROVED** [review]
Hi, thanks for the PR. The change looks good. Need to update documentation but I will do that in any case later this month. Don't hesitate to send new PRs if you want to change something. Great job!

File: serverless/openvino/omz/public/mask_rcnn_inception_resnet_v2_atrous_coco/nuclio/function.yaml (hunk 1 above)
Comment (xuchenCN) [reply serverless/openvino/omz/public/mask_rcnn_inception_resnet_v2_atrous_coco/nuclio/function.yaml:5]: Error - Service "nuclio-openvino-omz-public-mask-rcnn-inception-resnet-v2-atrous-coco" is invalid: metadata.name: Invalid value: "nuclio-openvino-omz-public-mask-rcnn-inception-resnet-v2-atrous-coco": must be no more than 63 characters

**Pull request merged**